easybuild/easyconfigs/s/SCOTCH/SCOTCH-6.0.4-intel-2018a.eb

```diff
@@ -0,0 +1,18 @@
+name = 'SCOTCH'
+version = '6.0.4'
+
+homepage = 'http://gforge.inria.fr/projects/scotch/'
+description = """Software package and libraries for sequential and parallel graph partitioning,
+static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://gforge.inria.fr/frs/download.php/file/34618/']
+sources = ['%(namelower)s_%(version)s.tar.gz']
```

Comment:
@SimonPinches `SHA256` checksum is missing here?

Reply:
Oops, added now.